main.py

```diff
@@ -48,10 +53,14 @@
         #perform the action
         #update agent and box locations
         #append path to history
-        #board_boxes
 
         #update Q values in Q Table
-        # q_table = Q.update_q_table(q_table, rewards)
+        # q_table = Q.update_q_table(q_table, rewards, new_box_position, action, discount_factor, learning_rate)
+
+        # terminal, status = Q.is_terminal_state(board, boxes)
 
-        # terminal = Q.is_terminal_state(board, boxes)
+    # if status == DEAD:
+    #     print(DEAD)
+    # if status == GOAL:
+    #     print(GOAL)
```

Comment:
Leave this in local branch--I don't think commented-out code should go in master